### DISCUSSION TITLE
Welcome to DeepHypergraph Discussions!

### DISCUSSION THREAD

**Discussion by yifanfeng97** [iMoonLab/DeepHypergraph#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Request to implement the features/models/layers you need.
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Li025** [comment]
I have benefited a lot from your work. Now I want to use the DHG you provided to realize a whole graph classification, but it seems that only node classification function is provided at present. May I ask if there is any convenient way for me to quickly realize this idea? If not, I hope your team can launch an api for the classification of the whole picture. Thank you very much.
